app/src/main/java/nl/tudelft/bbw/controller/CryptoController.java

```diff
@@ -58,4 +64,37 @@ public final String decryptString(String data) {
             throw new RuntimeException("Error - Could not decode string: " + e);
         }
     }
+
+    /**
+     * encryptFile method
```

Comment:
I have not seen it anywhere in the code I work with. Consistency is not a reason for bad comments.